### PR TITLE
feat(evmutil): add proto for MsgConvertNativeCoinToERC20

### PR DESCRIPTION
**Pull request by pirtleshell**
## Description
Adds the proto with Msg implementations and validations for ConvertNativeCoinToERC20 message. The message is registered in evmutil's message server to allow the binary to build, but the implementation will be added in a following PR. Changelog & doc updates will be made with the implementation.

Hoping to keep the PRs as bite-sized as possible in hopes of efficient and effective reviews.

## Checklist
 - [x] ~~Changelog has been updated as necessary.~~

### CODE REVIEW

File: x/evmutil/types/msg.go [hunk 1]
```diff
@@ -12,17 +12,26 @@ import (
 // ensure Msg interface compliance at compile time
 var (
 	_ sdk.Msg            = &MsgConvertCoinToERC20{}
-	_ sdk.Msg            = &MsgConvertERC20ToCoin{}
 	_ legacytx.LegacyMsg = &MsgConvertCoinToERC20{}
+	_ sdk.Msg            = &MsgConvertERC20ToCoin{}
 	_ legacytx.LegacyMsg = &MsgConvertERC20ToCoin{}
+
+	_ sdk.Msg            = &MsgConvertNativeCoinToERC20{}
```
Comment (galxy25) [inline x/evmutil/types/msg.go:19]: prefer `Cosmos` vs Native as conceptually Kava is a co-chain that "natively" supports evm and cosmos blockchains but in the future might also allow for converting things like BTC more directly

File: x/evmutil/keeper/msg_server.go [hunk 2]
```diff
@@ -21,6 +21,10 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 
 var _ types.MsgServer = msgServer{}
 
+////////////////////////////
+// EVM-native assets -> SDK
```
Comment (galxy25) [inline x/evmutil/keeper/msg_server.go:25]: pretend I don't know: what is this SDK you speak of?

**Review by galxy25 — APPROVED** [review]

File: x/evmutil/types/msg.go (hunk 1 above)
Comment (pirtleshell) [reply x/evmutil/types/msg.go:19]: i support the name change. it's big a big enough change set to move references from "Native Coin" to "Cosmos Coin" that i want to make it in a follow up pr that changes all the unreleased types (like the allowed_native_denoms param), not just the one for this message.

Comment (pirtleshell) [reply x/evmutil/types/msg.go:19]: https://github.com/Kava-Labs/kava/pull/1596

**Pull request merged**